lib/main.js

```diff
@@ -43,6 +43,11 @@ app.use('/micropub/:targetsite', micropub({
   tokenReference: function (req) {
     return (req.targetsite.token || []).concat(config.token);
   },
+  queryHandler: function (q, req) {
+    if (q === 'syndicate-to' && req.targetsite.syndicateTo) {
+      return { 'syndicate-to': req.targetsite.syndicateTo };
```

Comment:
I can take a stab, but since I'm not actually using this version yet, might not be able to test properly.